docs/usage.rst

```diff
@@ -1149,7 +1149,7 @@ this example:
   >>> from unyt import cm
   ...
   >>> data = [1, 2, 3]*cm
-  >>> np.save('my_data_cm.npy', data)
+  >>> np.save('my_data_cm.npy', data.d)
```

Comment:
Given this example exists mostly to show how unpractical `np.save` is with unyt_arrays, I this it's probably okay to add this small amount of additional complexity ?